src/main/kotlin/com/exactpro/th2/codec/AbstractCodecProcessor.kt

```diff
@@ -42,7 +43,9 @@ abstract class AbstractCodecProcessor(
     }
 
     protected fun String?.onErrorEvent(message: String, messagesIds: List<MessageID> = emptyList(), cause: Throwable? = null) {
-        logger.error(message, cause)
+        logger.error("$message. Message id's: ${messagesIds.joinToString(", ") { 
+            it.toDebugString()
```

Comment:
Maybe print them as `<session-alias>.<direction>.<sequence>[.<subsequence>]*` instead of JSON?

Reply:
@Nikita-Smirnov-Exactpro are u okay with it?

Reply:
I'm ok with @ivandruzhinin proposal